### PR TITLE
Static Analysis Tool Integration with CI

### PR DESCRIPTION
**Pull request by j-suguma**
Added the gradle task to run the static analysis tool after the build for Jenkins

### CODE REVIEW

**Comment by j-suguma** [comment]
Build passing:
image

Static Analysis:
image

File: app/build.gradle [hunk 1]
```diff
@@ -150,4 +159,21 @@ task copySecretsIfNotExists << {
...
+    classpath = files()
+    ignoreFailures = true
+
+    reports {
+        xml.enabled = true
+        html.enabled = false
+        html.destination = "$project.buildDir/outputs/findbugs/findbugs.xml"
+    }
+}
+
```
Comment (coreylevine) [inline app/build.gradle]: Remove extra blank line

**Comment by j-suguma** [comment]
You can then click the type and see exactly what line the "error" is and hovering over it says what it should be.
image

image

**Comment by coreylevine** [comment]
@j-suguma Thanks! So now we should update our PR policy. I think from now on we should only approve PRs if the newly added code is approved by the static analysis tool.

**Comment by j-suguma** [comment]
@rekandiah i put the build passing above in my first comment with pictures. I just didnt run tests because i didnt add any new ones and only edited the gradle file not the source code!

**Pull request merged**